python/cuxfilter/charts/core/non_aggregate/core_graph.py

```diff
@@ -50,7 +50,7 @@ def edge_columns(self):
 
     @property
     def node_columns(self):
```

Comment:
I would rewrite this to avoid writing the first three column names twice.
```python
def node_columns(self):
    cols = [
        self.node_id,
        self.node_x,
        self.node_y,
    ]
    if self.node_aggregate_col != self.node_id:
        cols.append(self.node_aggregate_col)
    return cols
```

Reply:
Yep, agreed, this is better, applied